README.md

```diff
@@ -0,0 +1,29 @@
+# MADiE Measure Service
+
+This is a Spring Boot micro-service which is responsible for operations associated with the MADiE Measure Micro Frontend.
+
+# Running Locally
+
+The application can be run as a spring-boot application. It requires a locally running MongoDb instance.
+
+The MongoDb instance can be run with 
+
+```
+docker compose down --remove-orphans && docker volume prune && docker compose build --no-cache && docker compose up --force-recreate --build madie-mong
```

Comment:
missing "o" at the end of the line for "madie-mongo" - command will likely fail without that